src/controls/NumericControls/NumericSliderControl.test.tsx

```diff
@@ -0,0 +1,98 @@
+import { describe, expect, test, it } from "vitest"
+import { screen, waitFor } from "@testing-library/react"
+import { userEvent } from "@testing-library/user-event"
+import { render } from "../../common/test-render"
+import { JSONSchema } from "json-schema-to-ts"
+import {
+  numericSliderBasisPointsSchema,
+  numericSliderUISchema,
+  numericSliderFinalGradeSchema,
+  numericSliderPercentageUISchema,
+  numericSliderUISchemaWithRule,
+} from "../../testSchemas/numericSchema/numericSliderSchema"
+
+describe("NumericSliderControl", () => {
+  test("renders a slider and number input with no UISchema provided", () => {
+    render({
+      schema: numericSliderBasisPointsSchema,
+    })
+
+    expect(screen.getByRole("slider")).not.toBeNull()
+    expect(screen.getByText("Basis Points")).not.toBeNull()
```

Comment:
I think these will be linting errors once main is merged into your branch: `prefer-implicit-assert` linting rule means getBy* and findBy* are preferred on their own bc getBy* throws an error if it doesn't find anything and findBy* promise rejects if it doesn't find anything

Reply:
✅